src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
The class `BalanceOperation` should not begin with an empty line. Please remove the redundant empty line at the start of the class implementation.